sympy/polys/tests/test_ring_series.py

```diff
@@ -154,13 +154,31 @@ def test_log():
     p = 1 + x
     p1 = rs_log(p, x, 4)/x**2
     assert p1 == 1/3*x - 1/2 + x**(-1)
+
     p = 1 + x +2*x**2/3
     p1 = rs_log(p, x, 9)
     assert p1 == -17*x**8/648 + 13*x**7/189 - 11*x**6/162 - x**5/45 + \
-      7*x**4/36 - x**3/3 + x**2/6 + x
-    p2 = rs_series_inversion(p, x, 9)
-    p3 = rs_log(p2, x, 9)
-    assert p3 == -p1
+                   7*x**4/36 - x**3/3 + x**2/6 + x
+
+    p = x + 2
+    c = _get_constant_term(p,x)
+    c_ex = c.as_expr()
+    R = R.add_gens([log(c_ex)])
+    p1 = rs_log(x+2,x,4)
+    p2 = p1 - R(log(c_ex))
```

Comment:
@Shaswat27 You need to write tests for `rs_series`. Something like:
`rs_series(log(x+1), x, 5)`

Moreover, the constant term is supposed to be handled by the code of `rs_log` and not manually.